Makefile

```diff
@@ -18,7 +18,15 @@ TEST_FILES=$(wildcard *_test.go)
 TELEMETRY_TEST_DIR = build/tests/gnmi_server
 TELEMETRY_TEST_BIN = $(TELEMETRY_TEST_DIR)/server.test
 ifeq ($(ENABLE_TRANSLIB_WRITE),y)
-BLD_FLAGS := -tags gnmi_translib_write
+  ifeq ($(ENABLE_NATIVE_WRITE),y)
```

Comment:
>ENABLE_NATIVE_WRITE [](http://example.com/codeflow?start=10&length=19)

You may check each ENABLE_XXX flag, and append to string, and finally set BLD_FLAGS by the string. Current nested if-else is not scalable for future extra flags.